### PR TITLE
Adds new community project

### PR DESCRIPTION
**Pull request by M0rdecay**
### Description
Adds new community project
 
### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

**Review by dtaivpp — APPROVED** [review]
Looks good to me and I can see the link to OpenSearch

File: _community_projects/gekatateam-neptunus.markdown [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+name: Neptunus
+description: Neptunus is written in Go data processing pipelines engine for consuming, transforming and producing events that can be used as a Logstash alternative.
+owner: Gekatateam
+link: https://github.com/gekatateam/neptunus
+license: Apache 2.0
+license_link: https://github.com/gekatateam/neptunus/blob/main/LICENCE
+owner_link: https://github.com/organizations/gekatateam
```
Comment (krisfreedain) [inline _community_projects/gekatateam-neptunus.markdown:8]: "https://github.com/organizations/gekatateam" doesn't seem to be a valid link - should it be "https://github.com/gekatateam" instead? Please let us know and update @M0rdecay - thanks

**Review by krisfreedain — CHANGES_REQUESTED** [review]
small link fix needed

File: _community_projects/gekatateam-neptunus.markdown (hunk 1 above)
Comment (M0rdecay) [reply _community_projects/gekatateam-neptunus.markdown]: Yes, you're right! Fixed

Comment (krisfreedain) [reply _community_projects/gekatateam-neptunus.markdown]: thanks @M0rdecay

**Review by krisfreedain — APPROVED** [review]

**Pull request merged**